internal/client/tailscale/tailscale.go

```diff
@@ -50,3 +57,26 @@ func NewClient(tailnet string, auth AuthMethod) *Client {
 type Client struct {
 	*tsclient.Client
 }
+
+// HandleErrorResponse is an alias to tailscale.com/client/tailscale.
+func HandleErrorResponse(b []byte, resp *http.Response) error {
+	return tsclient.HandleErrorResponse(b, resp)
+}
+
+// SendRequest add the authentication key to the request and sends it. It
+// receives the response and reads up to 10MB of it.
+func SendRequest(c *Client, req *http.Request) ([]byte, *http.Response, error) {
+	resp, err := c.Do(req)
+	if err != nil {
+		return nil, resp, err
+	}
+	defer resp.Body.Close()
+
+	// Read response. Limit the response to 10MB.
```

Comment:
It would be helpful to include more "why" than "what" in this comment. It doesn't seem unreasonable as a failsafe, but as a reader I'm left with a lot of questions like how likely this is, are there relevant checks server-side, why 10MB, will users be able to recover from this scenario etc.

Reply:
All good questions. I just copied this from [here](https://github.com/tailscale/tailscale/blob/main/client/tailscale/tailscale.go#L131). I'll see if I can put some more meat around the comment...